### ISSUE TITLE
Public questions get blank response on some devices

### ISSUE THREAD

**Issue by neoecos** [metabase/metabase#10068]
### Bugs

In some cases a metabase public question link, loads with empty data. Not refreshing will solve the issue. 

![WhatsApp Image 2019-05-30 at 12 16 32](https://user-images.githubusercontent.com/1490677/58651065-f4e7fd00-82d5-11e9-8002-b64713c7c20d.jpeg)

- Browser: Safari iOS 12, Andriod Chrome
- OS: iOS, Android 8
- DB: Postgresql
- Metabase 0.32.8 hosted in AWS ElasticBeansTalk with java 8
-Metabase db: Postgresql

- *Repeatable steps to reproduce the issue*

I haven't yet identified the steps to reproduce,  but the log is the following when this happens:
```

may 30 17:15:41 DEBUG metabase.middleware.log :: GET /api/public/card/9d4197ce-9bb1-43de-ac50-f6bef21934f9 200 2 ms (1 DB calls) Jetty threads: 4/50 (14 idle, 0 queued) (378 total active threads) Queries in flight: 0
may 30 17:15:41 DEBUG metabase.middleware.log :: GET /api/session/properties 200 2 ms (1 DB calls) Jetty threads: 4/50 (14 idle, 0 queued) (378 total active threads) Queries in flight: 0
may 30 17:15:41 DEBUG metabase.middleware.log :: GET /api/user/current 401 65 µs (0 DB calls) 
"Unauthenticated"

may 30 17:15:41 DEBUG metabase.middleware.log :: GET /api/public/card/9d4197ce-9bb1-43de-ac50-f6bef21934f9/query 200 [ASYNC: completed] 64 ms (9 DB calls) Jetty threads: 2/50 (15 idle, 0 queued) (378 total active threads) Queries in flight: 0
```

-------

**Comment by flamber** [comment]
Hi @neoecos 
Does this only happen to Table visualizations?
I'm not sure what you mean by "Not refreshing will solve the issue"?

@paulrosenzweig This looks like the same issue as you just fixed in #10053

**Comment by neoecos** [comment]
@flamber 'refreshing the browser page'

**Comment by neoecos** [comment]
@flamber @paulrosenzweig this has also happened in Android, using default browser and Chrome.

**Comment by paulrosenzweig** [comment]
Hmm in that case it sounds like it's different from the one fixed in #10053. That was persistent across refreshes. From the screenshot, this does look identical though. 

@neoecos if you keep refreshing does it ever happen again? Or, does it continue to work after that first page load? Have you ever seen this issue on a non-mobile device?

**Comment by neoecos** [comment]
@paulrosenzweig no, if we keep refreshing it does not work again. It never happened it non-mobile browsers. But it happend in Android and iOS and is very common.

**Comment by neoecos** [comment]
This issue looks is only happening in Webkit based mobile browsers, i asked people in which devices is not working to use Firefox Focus on mobile, and it's working.

I would really appreciate your help, even though it is 'downgrading' metabase until a fix is done.

**Comment by paulrosenzweig** [comment]
After looking into it more, I'm pretty sure @flamber is right and this was solved with #10053. I just checkout out 0.32.8 and reproduced this on mobile Safari repeatedly. On master however, it seems to work fine.

Given that, I'll close this issue. If you're still experiencing this after the next release, please reopen it and let us know.
